test/spec/adloader_spec.js

```diff
@@ -92,4 +94,11 @@ describe('adLoader', function () {
     expect(script.z).to.equal('A');
     expect(script.y).to.equal(2);
   });
+
+  it('should disable loading external script for activity rule set', function () {
+    const unregisterRule = registerActivityControl(LOAD_EXTERNAL_SCRIPT, 'loadExternalScript config', () => ({allow: false}));
+    adLoader.loadExternalScript('someURL2', 'debugging');
+    expect(utilsLogErrorStub.called).to.be.true;
+    unregisterRule();
```

Comment:
Test teardown is better done in a teardown hook (`after`/`afterEach`), or in a `finally` block. This is fine if the test passes, but if at some point some code changes makes it fail, the teardown would not run and can potentially affect other tests, which makes it more difficult to find the failure.